### PR TITLE
BAEL-4751: Java 14 New Features

### PR DESCRIPTION
**Pull request by sampada07**

### CODE REVIEW

File: core-java-modules/core-java-14/src/test/java/com/baeldung/java14/newfeatues/SwitchExprUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+        assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(() -> isTodayHolidayInJava8("SOMEDAY"));
+
+        assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(() -> isTodayHolidayInJava14("SOMEDAY"));
+    }
+
+    private boolean isTodayHolidayInJava8(String day) {
+
+        boolean isTodayHoliday;
+        switch (day) {
```
Comment (eric-martin) [inline core-java-modules/core-java-14/src/test/java/com/baeldung/java14/newfeatues/SwitchExprUnitTest.java:24]: indent the body of the switch

File: core-java-modules/core-java-14/src/test/java/com/baeldung/java14/newfeatues/SwitchExprUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        default:
+            throw new IllegalArgumentException("What's a " + day);
+        }
+
+        return isTodayHoliday;
+
+    }
+
+    private boolean isTodayHolidayInJava14(String day) {
+        return switch (day) {
```
Comment (eric-martin) [inline core-java-modules/core-java-14/src/test/java/com/baeldung/java14/newfeatues/SwitchExprUnitTest.java:45]: indent the body of the switch

File: core-java-modules/core-java-14/src/test/java/com/baeldung/java14/newfeatues/SwitchExprUnitTest.java (hunk 1 above)
Comment (sampada07) [reply core-java-modules/core-java-14/src/test/java/com/baeldung/java14/newfeatues/SwitchExprUnitTest.java:24]: Done!

File: core-java-modules/core-java-14/src/test/java/com/baeldung/java14/newfeatues/SwitchExprUnitTest.java (hunk 2 above)
Comment (sampada07) [reply core-java-modules/core-java-14/src/test/java/com/baeldung/java14/newfeatues/SwitchExprUnitTest.java:43]: Done!

**Pull request merged**